src/java/org/apache/cassandra/tools/SSTableMetadataViewer.java

```diff
@@ -447,6 +470,19 @@ private void field(String field, Object value, String comment)
         }
         this.out.println(sb.toString());
     }
+    public static String durationString(Long value)
+    {
+
+        TimeUnit tsUnit = TimeUnit.MICROSECONDS;
```

Comment:
`tsUnit` is not used in this method, probably a leftover

`duration` can be inlined (`return "Days ..."`)